internal/isoutil/isoutil.go

```diff
@@ -27,6 +30,11 @@ type installerHandler struct {
 	workDir string
 }
 
+type OffsetInfo struct {
+	Key    [8]byte
+	Offset int64
```

Comment:
Consider encapsulating the multiple instances of this struct in the ISO file into one struct, and also have a version number in there - so backwards compatibility is easier to add in the future

Reply:
Hmm, I'm not sure backwards compatibility should indeed be an issue here. I.e. the service generates the template, and we should ensure that we store the latest version and act accordingly. I mean, when changing the template, we delete the old version and upload the a fresh in upon service startup, so new multiple version support for now. Any way, we could probably tackle it later on if we decide that indeed an issue.